Get-NetView.psm1

```diff
@@ -28,19 +28,6 @@ $ExecFunctions = {
         Write-Output $Message | Out-File -Encoding ascii -Append $out
     } # ExecControlError()
 
-    function ExecCommandText {
-        [CmdletBinding()]
-        Param(
-            [parameter(Mandatory=$true)] [String] $Command
-        )
-
-        # Mirror command execution context
-        Write-Output "$env:USERNAME @ ${env:COMPUTERNAME}:"
-
-        # Mirror command to execute
-        Write-Output "$(prompt)$Command"
-    } # ExecCommandText()
-
```

Comment:
Please describe why this is being removed.  This means command output to screen will not be displayed.  IMHO, it will make development and debugging much harder since we will have to inspect the output fies.

Reply:
It is in the change log:
> Inline ExecCommandText to simplify the script slightly.

Also, this is imported into every single thread context, so there will be marginal perf gains from importing/calling one less function